tests/enable_saved_search_conftest.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, splunk):
 
     def wait_for_lookup(self, lookup):
         splunk_client = client.connect(**self.splunk)
-        for _ in range(30):
+        for _ in range(60):
```

Comment:
If we change from 30 to 60 seconds are we sure it will be enough? Beacuse user has no way of changing this value.

Reply:
Yes 60 seconds timer is working fine and it will not affect users, it is used to validate function test of pytest-splunk-addon.